src/core/js/views/buttonsView.js

```diff
@@ -76,48 +72,51 @@ define(function() {
-				//disable submit button on correct (i.e. no model answer)
-                this.$('.buttons-action').a11y_cntrl_enabled(false);
-
-                if (!this.model.get("_canShowFeedback")) {
-                    if (!this.$el.is(".no-state")) {
-                        //if no feedback, complete correct and has state, force focus to component state
-                        _.defer(_.bind(function() {
-                            $("." + this.model.get("_id") + " .accessibility-state [tabindex]").focusNoScroll();
-                        }, this));
-                    }
-                }
+          //use correct instead of complete to signify button state
+          if (changedAttribute === 'correct') {
+            //disable submit button on correct (i.e. no model answer)
+            this.$('.buttons-action').a11y_cntrl_enabled(false);
+
+            if (!this.model.get("_canShowFeedback")) {
+              if (!this.$el.is(".no-state")) {
+                //if no feedback, complete correct and has state, force focus to component state
+                _.defer(_.bind(function() {
+                  $("." + this.model.get("_id") + " .accessibility-state [tabindex]").focusNoScroll();
+                }, this));
+              }
+            }
 
-            } else {
-                // Backwords compatibility with v1.x
-                var ariaLabel = this.model.get('_buttons')["_" + changedAttribute].ariaLabel;
-                var buttonText = this.model.get('_buttons')["_" + changedAttribute].buttonText;
-
-                switch (changedAttribute) {
-                    case "showCorrectAnswer": case "hideCorrectAnswer":
-    				    //make model answer button inaccessible but enabled for visual users
-    				    //	due to inability to represent selected incorrect/correct answers to a screen reader, may need revisiting
-                        this.$('.buttons-action').a11y_cntrl(false).html(buttonText).attr('aria-label', ariaLabel);
-                        break;
-                    default:
-    				    //enabled button, make accessible and update aria labels and text.
-                        this.$('.buttons-action').a11y_cntrl_enabled(true).html(buttonText).attr('aria-label', ariaLabel);
-                }
+          } else {
+            // Backwords compatibility with v1.x
+            var ariaLabel = this.model.get('_buttons')["_" + changedAttribute].ariaLabel;
+            var buttonText = this.model.get('_buttons')["_" + changedAttribute].buttonText;
+
+            switch (changedAttribute) {
+              case "submit":
+              case "showCorrectAnswer":
+              case "hideCorrectAnswer":
+                //make model answer button inaccessible but enabled for visual users
+                //	due to inability to represent selected incorrect/correct answers to a screen reader, may need revisiting
+                this.$('.buttons-action').a11y_cntrl_enabled(true).html(buttonText).attr('aria-label', ariaLabel);
```

Comment:
Has `a11y_cntrl(false)` been amended intentionally to `a11y_cntrl_enabled(true)`?

The existing code exists specifically to make the *Correct/My answer* buttons inaccessible to screenreaders whilst still remaining enabled.